tests/test_config.py

```
@@ -171,3 +171,48 @@ def test_max_response_chars_rejects_non_positive(bad_value):
+        config = ServerConfig()
+    assert config.stateless_http is False
+
+
+def test_stateless_http_from_cli():
+    """--stateless-http CLI argument controls stateless mode."""
+    config = CliApp.run(ServerConfig, cli_args=["--stateless-http", "false"])
+    assert config.stateless_http is False
+
+
+@pytest.mark.parametrize(
+    "transport,expected_in_kwargs",
```

Ruff (0.15.20)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)